hoot-services/src/main/java/hoot/services/HootServletContext.java

```diff
@@ -78,20 +77,16 @@ private static void initSLF4JBridgeHandler() {
         SLF4JBridgeHandler.install();
     }
 
-    private static void createIngestFolder() {
-        File ingestFolder = new File(TILE_SERVER_PATH);
-        if (!ingestFolder.exists()) {
-            if (!ingestFolder.mkdir()) {
-                throw new RuntimeException("Error creating " + ingestFolder.getAbsolutePath() + " directory!");
-            }
-        }
-    }
+    private static void createFolders() {
+        String[] folders = { UPLOAD_FOLDER, BASEMAPS_TILES_FOLDER, RPT_STORE_PATH,
+                BASEMAPS_FOLDER, SCRIPT_FOLDER, TEMP_OUTPUT_PATH };
 
-    private static void createUploadFolder() {
-        File uploadDir = new File(UPLOAD_FOLDER);
-        if (!uploadDir.exists()) {
-            if (!uploadDir.mkdir()) {
-                throw new RuntimeException("Error creating " + uploadDir.getAbsolutePath() + " directory!");
+        for (String folder : folders) {
+            try {
+                FileUtils.forceMkdir(new File(folder));
+            }
```

Comment:
Is there a way to handle migration of the contents of existing directories here?  And maybe clean up those old root level dirs?